update_fuel_dependencies.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Run this script to only bump fuel maintained dependencies. 
+#
+# We currently pin dependencies using "X.Y" in the root Cargo.toml file.
+# Since `cargo build` does not check for minor version bumps at each invocation
+# it is hard to move between new versions, especially for minor bumps which
+# happens much often. Use this script to keep every fuel owned dependency up to
+# date.
+
+# Define the list of fuel maintained crates
+crates=(
+    "fuel-abi-types"
+    "fuel-core-client"
+    "fuel-core-types"
+    "fuels"
+    "fuels-core"
+    "fuels-accounts"
+    "fuel-asm"
+    "fuel-crypto"
+    "fuel-types"
+    "fuel-tx"
+    "fuel-vm"
+    "forc-wallet"
+)
+
+# Run `cargo update -p <crate_name>` for each fuel owned crate. 
+for crate in "${crates[@]}"; do
+    echo "Updating package: $crate"
+    cargo update -p "$crate"
+done
```

Comment:
Weird... `cargo update` doesn't seem to document `-p` but it's the same as `--package <SPEC>`, but having a look `--package` isn't documented either in `cargo` 🙃

Reply:
It looks like `cargo update -p "$crate"` is equivalent to `cargo update "$crate"`

Reply:
Should we prefer using non -p version? I don't think it is important but it is a little weird that -p is not documented in the original documentation. I found out about this param in a mirror doc of cargo update hosted by some third party. 

I am not sure where cargo-update is hosted. I see a repo named cargo-update under non-rust lang organization 🤷 

I think this will be fine, and possibly not documented due to lack of documentation efforts in that area. Resolving this for now feel free to ping me if anyone thinks non -p version should be preferred here 🙌